tripal/tests/src/Functional/Services/TripalFieldCollectionTest.php

```diff
@@ -163,7 +162,7 @@ public function testTripalFieldCollection() {
     $this->assertIsInt($defaults['cardinality'], 'The cardinality element is not an integer');
     $this->assertIsArray($defaults['storage_settings']['storage_plugin_settings'], 'The storage_plugin_settings element should be an array');
 
-    // Make sure the setFieldDefDefaults functions doesn't change any valid values
+    // Make sure the setFieldDefDefaults functions doesn't change any valid values.
```

Comment:
```suggestion
    // Make sure the setFieldDefDefaults function doesn't change any valid values.
```
Can't help fixing this